config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  get "static_pages/home"
-  get "static_pages/help"
-  get "static_pages/about"
   root "static_pages#home"
+  get "/help", to: "static_pages#help"
+  get "/about", to: "static_pages#about"
+  get "/contact", to: "static_pages#contact"
+  get "/signup",  to: "users#new"
```

Comment:
thì thấy bên to nó có 1 các controllẻ là static_page vs user đúng k thì đấy là 2 cái block giữa chúng thì cho thêm 1 dòng